src/components/changePlanner/CPDetailView.tsx

```diff
@@ -358,157 +392,196 @@ class CPDetailView extends React.Component<
+                              {this.renderAssetData(
+                                modification.asset,
+                                modification
+                              )}
+                            </Pre>
+                            <Pre>
+                              <h3>Change Plan Asset</h3>
+                              {modification.asset_cp
+                                ? this.renderAssetData(
                                     modification.asset_cp,
                                     modification
                                   )
-                                  : null}
-                              </Pre>
-                            </div>
-                          ) : (
-                              <div className="cp-details">
-                                <Pre>
-                                  <h3>Change Plan Asset</h3>
-                                  {modification.asset_cp
-                                    ? this.renderAssetData(
-                                      modification.asset_cp,
-                                      modification
-                                    )
-                                    : null}
-                                </Pre>
-                              </div>
-                            )}
-                        </div>
-                      </Collapse>
-                    </li>
-                  );
+                                : null}
+                            </Pre>
+                          </div>
+                        ) : (
+                          <div className="cp-details">
+                            <Pre>
+                              <h3>Change Plan Asset</h3>
+                              {modification.asset_cp
+                                ? this.renderAssetData(
+                                    modification.asset_cp,
+                                    modification
+                                  )
+                                : null}
+                            </Pre>
+                          </div>
+                        )}
+                      </div>
+                    </Collapse>
+                  </li>
+                );
+              }
+            )
+          ) : (
+            <Callout title="No modifications for this change plan"> </Callout>
+          )}
+        </ul>
+        <div className="detail-buttons-wrapper">
+          <div className={"detail-buttons-cp"}>
+            <div>
+              <AnchorButton
+                disabled={this.disableExecute()}
+                intent="none"
+                icon="document-open"
+                text="Generate Work Order"
+              />
+            </div>
+            <div className="cp-dividers">
+
+              <Divider className="cp-detail-divider" />
+              <p className={Classes.DARK + " cp-detail-divider-text"}>then</p>
+              <Divider className="cp-detail-divider" />
```

Comment:
I like this change but it's sort of hard to see - would it be hard to make the line separator darker or maybe highlight the whole thing on a darker card background?

Reply:
yes can make the separator lighter